### PR TITLE
Add watch operation [US #95][Task #98]

### PR DESCRIPTION
**Pull request by todd829**
### WHAT kind of change does this PR introduce?
Operations can now be completed on the calculator on the android wear interface.

### HOW is this accomplished?
* By adding a `GestureDetector` which displays the operation screen
* Smart clear function is working on the wear ui 
* Binding of the buttons to the `CalculatorImpl` class from the commons package
...

### Checklist
- [x] Docs have been added / updated to reflect the changes
- [x] I have reviewed and tested the changes :white_check_mark:
- [x] I have squashed my commits to have a reasonable amount of commits
- [x] All of the code I have written adhere to the coding conventions outlined in the coding conventions section of the wiki

### CODE REVIEW

File: wear/src/main/java/com/simpletools/calculator/wear/MainActivity.kt [hunk 1]
```diff
@@ -16,6 +19,28 @@ class MainActivity : WearableActivity(), Calculator {
...
+
+
+    class SimpleGestureListener(val activity: MainActivity) : GestureDetector.SimpleOnGestureListener() {
+
+        override fun onFling(e1: MotionEvent?, e2: MotionEvent?, velocityX: Float, velocityY: Float): Boolean {
+            if(Math.abs(velocityY) > Math.abs(velocityX)){
+                if (e2!!.y > e1!!.y) {
+                    Log.d("debug","DOWN")
+                    activity.showOperations()
+                    // direction up
```
[the remark this answers is not in the document]
Comment (todd829) [reply wear/src/main/java/com/simpletools/calculator/wear/MainActivity.kt]: It's from the top to the bottom so we see it as a down swipe

[the remark this answers is not in the document]
Comment (miisaka) [reply wear/src/main/java/com/simpletools/calculator/wear/MainActivity.kt]: Alright makes more sense

File: wear/src/main/java/com/simpletools/calculator/wear/MainActivity.kt [hunk 2]
```diff
@@ -55,6 +87,30 @@ class MainActivity : WearableActivity(), Calculator {
         setAmbientEnabled()
     }
 
+    fun showOperations(){
```
Comment (almiche) [inline wear/src/main/java/com/simpletools/calculator/wear/MainActivity.kt]: Smart solution!

**Comment by almiche** [comment]
✅ Styling is adequate and follows coding conventions outlined by the Kotlin Foundation

✅ New methods and functionalities have appropriate tests (testing edge cases + good coverage)

✅ Branch has an acceptable amount of commits with descriptive names and

✅ PR description matches the code outlined a has mentions to issues

✅ This PR attempts to implement a specific feature (no code blob or epic pr)

Status: **Approved**

Comments: Well done all looks well!
